tox.ini

```diff
@@ -12,7 +12,9 @@ passenv=HOME
 deps = pipenv
 basepython = python
 commands =
-    pipenv install --dev --ignore-pipfile
+    pipenv --three
```

Comment:
You should not need this, tox will setup an env with the correct python version based on it's config.